Source/DafnyLanguageServer.Test/Lookup/HoverTest.cs

```diff
@@ -553,5 +553,23 @@ function f(i: int): (r: int)
       Assert.AreEqual(MarkupKind.Markdown, markup.Kind);
       Assert.AreEqual("```dafny\nr: Pos\n```", markup.Value);
     }
+
+    [TestMethod]
+    public async Task HoverIngResultTypeShouldNotCrash() {
+      var source = @"
+datatype Position = Position(Line: nat)
+function ToRelativeIndependent(): (p: Position)
```

Comment:
@camrein I spotted another duplication of symbol registration in the type Position above.
The reason is, the function type is Position but the Result is a variable with a declared type, and the two types are the same, but one was inferred (the function type) and it's not visible.
Right now I'm catching all these errors as I'm developing Dafny, but it would be better to just ignore them.